user_docs/en/userGuide.t2t

```diff
@@ -1074,6 +1074,27 @@ When you scroll the display, you will then be able to read the rest of the word.
 
 Enabling this may allow for more fluent reading, but generally requires you to scroll the display more.
 
+==== Focus context presentation ====
+This option allows you to choose what context information NVDA will show on the braille display when an object gets focus.
+Context information (or focus ancestry) refers to the hierarchical tree of objects containing the focus object.

```

Comment:
I'm concerned about terminology overload here. Perhaps simplify to something like:

> `Context information (or focus ancestry) refers to the hierarchy of objects containing the focus.`